src/test/java/coresearch/cvurl/io/sse/EventParserTest.java

```diff
@@ -0,0 +1,170 @@
+package coresearch.cvurl.io.sse;
```

Comment:
You have several tests with the same structure. I think you can write a parameterized test instead. Btw, JUnit 5 does not require the public modifier for classes and methods.